### PR TITLE
Update install_dependencies.md to include arm64

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
In this PR I add a code cell with the arm64 installer script.
Also, I removed the `-b` which bypasses the license. 
In the copy, it's mentioned to look at the agreement, so I think it's better to let the user see it, rather than bypass.
Can revert that though.

### CODE REVIEW

File: tutorial/install_dependencies.md [hunk 1]
```diff
@@ -44,15 +42,22 @@ as well and you can skip to the next section.
...
+             ```bash
+             bash Miniforge3-MacOSX-arm64.sh
+             ```
 
-        4. To verify that your installation worked, close your Terminal window
+        5. To verify that your installation worked, close your Terminal window
            and open a new one. You should see `(base)` to the left of your
            prompt.
-        5. Finally, initialize miniforge with the command below. This makes sure
+        6. Finally, initialize miniforge with the command below. This makes sure
```
Comment (lagru) [inline tutorial/install_dependencies.md]: This updated numbering seems off?

**Review by lagru — COMMENTED** [review]
Looks good except for a minor comment on numbering. Thanks!

File: tutorial/install_dependencies.md (hunk 1 above)
Comment (psobolewskiPhD) [reply tutorial/install_dependencies.md]: Huh, no idea, sorry I missed it. I think there is a chance it's auto fixed when the the docs get built, but maybe not.

**Review by lagru — APPROVED** [review]
Thanks @psobolewskiPhD!

**Pull request merged**